File: 프로그래머스/lv1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        final int sLengths = s.length();
        if (sLengths % 2 == 0) {
            answer = s.substring((sLengths - 1) / 2, sLengths / 2 + 1);
        } else {
            answer = s.substring(sLengths / 2, sLengths / 2 + 1);
        }
        return answer;
    }   
}

File: 프로그래머스/lv1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.py
def solution(s):
    answer = ''
    STRING_LENGTH = len(s)
    
    if STRING_LENGTH % 2 == 0:
        answer = s[(STRING_LENGTH - 1) // 2 : (STRING_LENGTH) // 2 + 1]
    else:
        answer = s[STRING_LENGTH // 2]
    return answer